apps/io-func-wallet-solution-backend/src/infra/http/handlers/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { flow } from "fp-ts/function";
+import * as H from "@pagopa/handler-kit";
+
+export const successJwt = flow(
+  H.success,
+  H.withHeader("Content-Type", "application/entity-statement+jwt")
+);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick(naming):** ‏Not just a successful JWT response, but a specific JWT (an OIDC Federation entity statement).

We can even avoid finding a proper name, just ad the header at the response:
```ts
RTE.map(H.successJson),
RTE.map(H.withHeader("Content-Type", "application/entity-statement+jwt")),
```

Reply:
I reckon it's 200 even for POST! Same as [token endpoint](https://docs.italia.it/italia/spid/spid-cie-oidc-docs/it/versione-corrente/token_endpoint.html)!